### PR TITLE
Update cluster stats page graphs

### PR DESCRIPTION
**Pull request by venkateshamatam**
# Description
Update cluster stats page graphs and update usagestats.csv format to store logsBytes and metricsBytes
# Testing
- Tested graph values by ingesting over time
- No errors or warnings on browser console

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [ ] I have self-reviewed this PR.
- [ ] I have removed all print-debugging and commented-out code that should not be merged.
- [ ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

**Review by nkunal — COMMENTED** [review]
lgtm

File: pkg/usageStats/stats.go [hunk 1]
```diff
@@ -347,41 +353,47 @@ func FlushStatsToFile(orgid uint64) error {
...
+func UpdateStats(logsBytesCount uint64, logLinesCount uint64, orgid uint64) {
 	if _, ok := ustats[orgid]; !ok {
 		ustats[orgid] = &Stats{
 			BytesCount:         0,
 			LogLinesCount:      0,
 			TotalBytesCount:    0,
 			TotalLogLinesCount: 0,
+			LogsBytesCount:     0,
 		}
 	}
```
Comment (AndrewHess) [inline pkg/usageStats/stats.go:374]: nit: You can change the inside to just `ustats[orgid] = &Stats{}`; all the `Stats` fields are uint64, so they'll be 0 by default. Currently I find it a little odd that you're initializing some fields but not others (but Go initializes those others to 0 anyways)

Same comment for lines 384-389

File: static/js/cluster-stats.js [hunk 2]
```diff
@@ -72,14 +72,12 @@ function renderChart() {
         dataType: 'json',
         data: JSON.stringify(data)
     })
-        .then((res)=> {
-            $('#app-content-area').show();
-            drawStatsChart(res,data)
-        })
-        .catch(showCStatsError);
+    .then(res => drawStatsChart(res, data, "logs").then(() => drawStatsChart(res, data, "metrics")))
+    .catch(showCStatsError);
```
Comment (AndrewHess) [inline static/js/cluster-stats.js:76]: Why do you no longer need `$('#app-content-area').show();`? (I don't know what this does)

**Review by AndrewHess — APPROVED** [review]

File: static/js/cluster-stats.js (hunk 2 above)
Comment (venkateshamatam) [reply static/js/cluster-stats.js]: `$('#app-content-area').show();` is used to show the cluster stats div, and it was already being shown by default. I'm not very familiar with the UI code and I don’t know if there might be a case where this is required, so I added it back.

**Pull request merged**